en/docs/guides/login/oidc-parameters-in-auth-request.md

```diff
@@ -0,0 +1,149 @@
+    
+    -   none         `
+    -   login         `
+    -   consent
+    
+###prompt=none
+The silent authentication can be initiated by using the prompt=none parameter with the authentication request.
+
+```tab="Sample Request"
+https://<host>:9443/oauth2/authorize?response_type=token&client_id=NgTICXFPYnt7ETUm6Fc8NMU8K38a&redirect_uri=<callback_url>&prompt=none&scope=openid
+```
+
+If the user has an already authenticated session and a pre-configured consent with the WSO2 Identity Server a successful response can be obtained as follows.
+
+Successful Response
+
+```tab="Sample Response"
+https://<callback_url>#token_type=Bearer&expires_in=60&access_token=10a361a99aa4bd6e0aa79c6ea7bcdb66
+```
+
+Error Response
+
+```tab="Sample Response"
+   https://callback_url
+   error_description=ERROR_DESCRIPTION&
+   error=ERROR_CODE&
+   session_state==...
+```
+    
+   | Error                 | Error Description         | 
+   | --------------------- | ------------- | 
+   | login_required | Occurs when the user does not have a login session  |                            
+   | consent_required           | Occurs when the user has a login session but does not have a pre-configured consent  |                              
+   
+----
+
+
+###prompt=login
+The force authentication can be initiated by using the prompt=login parameter with the authentication request.
+
+```tab="Sample Request"
+https://<host>:9443/oauth2/authorize?response_type=token&client_id=NgTICXFPYnt7ETUm6Fc8NMU8K38a&redirect_uri=http://localhost:8080/playground2/oauth2client&prompt=none&scope=openid
+```
+
+If the user is successfully re-authenticated with the WSO2 Identity Server a successful  response can be obtained as follows.
+
+Successful Response
+
+```tab="Sample Response"
+https://<callback_url>#token_type=Bearer&expires_in=60&access_token=10a361a99aa4bd6e0aa79c6ea7bcdb66
+```
+
+Error Response
+
+```tab="Sample Response"
+   https://callback_url
+   error_description=ERROR_DESCRIPTION&
+   error=ERROR_CODE&
+   session_state==...
+```
+
+| Error                 | Error Description         | 
+   | --------------------- | ------------- | 
+   | login_required | Occurs when the user can not re-authenticate the user  |                            
+   
+   ----
+   
+###prompt=consent
+ Prompting forceful user consent can be initiated by using the prompt=consent parameter with the authentication request.
+ 
+ ```tab="Sample Request"
+ https://<host>:9443/oauth2/authorize?response_type=token&client_id=NgTICXFPYnt7ETUm6Fc8NMU8K38a&redirect_uri=http://localhost:8080/playground2/oauth2client&prompt=consent&scope=openid&access_token=10a361a99aa4bd6e0aa79c6ea7bcdb66
+ ```
+ 
+ If the user is successfully provided the consent again, even if the consent is already given,  the WSO2 Identity Server will return a successful  response as follows.
+
+Successful Response
+ 
+ ```tab="Sample Response"
```

Comment:
shall we add the tab title as "Successful Response" instead of "sample response" ?